gengokit/middlewares/middlewares.go

```diff
@@ -0,0 +1,63 @@
+// Package middlewares renders the service and endpointe middleware files in NAME-service/middlewares/.
+package middlewares
+
+import (
+	"io"
+
+	"github.com/pkg/errors"
+
+	"github.com/TuneLab/go-truss/gengokit"
+	"github.com/TuneLab/go-truss/gengokit/middlewares/templates"
+)
+
+// EndpointsPath is the path to the Endpoints middleware file that package middlewares renders
+const EndpointsPath = "NAME-service/middlewares/endpoints.gotemplate"
+
+// ServicePath is the path to the Service middleware file that package middlewares renders
+const ServicePath = "NAME-service/middlewares/service.gotemplate"
+
+// New returns a Middleware which can render EndpointsFile and ServiceFile as
+// well as read in previous versions of each respective file
+func New() *Middlewares {
```

Comment:
This seems kind of pointless unless the function actually does something. This could be replaced with

```
foo := new(middlewares.Middlewares)
```